src/store/hooks.ts

```diff
@@ -0,0 +1,5 @@
+import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux';
+import type { RootState, AppDispatch } from './index';
+
+export const useAppDispatch = () => useDispatch<AppDispatch>();
+export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;
```

Comment:
you don't need to create separate own functions for `useDispatch` and `useSelector`, you can use them directly imported from `react-redux` in any component